lambdas/service/app.py

```diff
@@ -771,9 +771,16 @@ def _dss_files(uuid, fetch=True):
     url = config.dss_endpoint + '/files/' + urllib.parse.quote(uuid, safe='')
     file_name = params.pop('fileName', None)
     wait = params.pop('wait', None)
+    request_number = params.pop('requestNumber', None)
```

Comment:
```suggestion
    request_index = params.pop('requestIndex', 0)
```

There is precedent for this naming in the manifest generation code.

We prefer 0-based.